lib/activerecord-import/import.rb

```diff
@@ -367,7 +367,7 @@ def import_helper( *args )
           column_names = self.column_names.dup
         end
 
-        if column_names.include?(primary_key) && columns_hash[primary_key].type == :uuid
+        if models.first.id.nil? && column_names.include?(primary_key) && columns_hash[primary_key].type == :uuid
```

Comment:
The `id` method in on an ActiveRecord model always returns the primary key whether it is named 'id' or something else.

I'm not sure specifying some ID's but not all would work. You would end up attempting to insert NULL ID values which would result in a SQL error.